Havit.Blazor.Components.Web.Bootstrap/Collapse/HxCollapse.razor

```diff
@@ -1,5 +1,5 @@
 ﻿@namespace Havit.Blazor.Components.Web.Bootstrap
 
-<div @ref="collapseHtmlElement" id="@this.Id" class="@GetCssClass()" data-bs-parent="@this.Parent" @attributes="this.AdditionalAttributes">
+<div @ref="collapseHtmlElement" aria-expanded="@InitiallyExpanded" id="@this.Id" class="@GetCssClass()" data-bs-parent="@this.Parent" @attributes="this.AdditionalAttributes">
```

Comment:
I left it this way because JS overrides the `aria-expanded` attribute anyways but I agree this doesn't look good and may cause issues, so I'll adjust it.

Reply:
I think JS changes the value when any ACTION in UI occurs (JS sets new attribute value), but if Blazor re-renders the component on its own, it won't be "fixed" by JS. See similar issues:
https://github.com/havit/Havit.Blazor/issues/19
https://github.com/havit/Havit.Blazor/issues/182